base/applications/notepad/lang/ro-RO.rc

```diff
@@ -33,46 +33,46 @@ MAIN_MENU MENU
 BEGIN
     POPUP "&Fișier"
     BEGIN
-        MENUITEM "&Nou\tCtrl+N", CMD_NEW
+        MENUITEM "No&u\tCtrl+N", CMD_NEW
         MENUITEM "Fereastră &nouă\tCtrl+Shift+N", CMD_NEW_WINDOW
-        MENUITEM "&Deschidere…\tCtrl+O", CMD_OPEN
-        MENUITEM "S&alvează\tCtrl+S", CMD_SAVE
-        MENUITEM "Sal&vare în…", CMD_SAVE_AS
+        MENUITEM "&Deschidere...\tCtrl+O", CMD_OPEN
+        MENUITEM "&Salvare\tCtrl+S", CMD_SAVE
+        MENUITEM "S&alvare ca...", CMD_SAVE_AS
         MENUITEM SEPARATOR
-        MENUITEM "Inițiali&zare pagină…", CMD_PAGE_SETUP
-        MENUITEM "I&mprimare…\tCtrl+P", CMD_PRINT
+        MENUITEM "Iniţiali&zare pagină...", CMD_PAGE_SETUP
+        MENUITEM "I&mprimare...\tCtrl+P", CMD_PRINT
         MENUITEM SEPARATOR
-        MENUITEM "I&eșire", CMD_EXIT
+        MENUITEM "&Ieşire", CMD_EXIT
     END
     POPUP "&Editare"
     BEGIN
-        MENUITEM "Anula&re\tCtrl+Z", CMD_UNDO
+        MENUITEM "&Anulare\tCtrl+Z", CMD_UNDO
         MENUITEM SEPARATOR
-        MENUITEM "&Decupează\tCtrl+X", CMD_CUT
-        MENUITEM "&Copiază\tCtrl+C", CMD_COPY
-        MENUITEM "&Lipește\tCtrl+V", CMD_PASTE
-        MENUITEM "Șt&erge\tDel", CMD_DELETE
+        MENUITEM "&Decupare\tCtrl+X", CMD_CUT
+        MENUITEM "&Copiere\tCtrl+C", CMD_COPY
+        MENUITEM "Li&pire\tCtrl+V", CMD_PASTE
+        MENUITEM "Şt&ergere\tDel", CMD_DELETE
         MENUITEM SEPARATOR
-        MENUITEM "&Găsire…\tCtrl+F", CMD_SEARCH
+        MENUITEM "Cău&tare...\tCtrl+F", CMD_SEARCH
         MENUITEM "&Următorul găsit\tF3", CMD_SEARCH_NEXT
-        MENUITEM "Î&nlocuire…\tCtrl+H", CMD_REPLACE
-        MENUITEM "S&alt la…\tCtrl+G", CMD_GOTO
+        MENUITEM "Î&nlocui&re...\tCtrl+H", CMD_REPLACE
+        MENUITEM "Salt &la...\tCtrl+G", CMD_GOTO
         MENUITEM SEPARATOR
-        MENUITEM "Sele&ctare totală\tCtrl+A", CMD_SELECT_ALL
-        MENUITEM "&Dată/Oră\tF5", CMD_TIME_DATE
+        MENUITEM "&Selectare totală\tCtrl+A", CMD_SELECT_ALL
+        MENUITEM "Dată/&Oră\tF5", CMD_TIME_DATE
     END
-    POPUP "F&ormatare"
+    POPUP "F&ormat"
     BEGIN
-        MENUITEM "În&cadrează liniile lungi", CMD_WRAP
-        MENUITEM "&Font…", CMD_FONT
+        MENUITEM "&Încadrare cuvânt", CMD_WRAP
+        MENUITEM "&Font...", CMD_FONT
```

Comment:
Why the change from "&Font…" to "&Font..." ? (1char to 3chars for the ...) The former is more compact horizontally, therefore is a good default choice (or whenever fearing that space is limited).
Same question for other places where you did that change.

Reply:
I saw thatg in English there are three dots as three characters, too. The same in French, Hungarian, German and many other resources.

Reply:
Like I said, I do think the single-character-"…" is a sane choice whenever we are in a rc file which already contains multibyte-characters, which means typically almost all rcs except en-US.
Yes, MS doesn't do that, but I see no urgent need to match MS exactly in that specific detail.
Often the single-character-"…" allowed us to make a line fit to a rather narrow button. So changing an existing single-character-"…" to the regular 3 dots is something that I would generally discourage.

Yes, my personal recommendation is to stick with the single-character-"…" !

Reply:
I changed it back. Now it's just one character.

Reply:
I understand. I changed it back, so now it's only one character.